### ISSUE TITLE
Deleting the super-legacy stuff

### ISSUE THREAD

**Issue by MrPowers** [awesome-spark/awesome-spark#199]
There are some references to super-legacy stuff in the README like [spark-csv](https://github.com/databricks/spark-csv) and [spark-mongodb](https://github.com/Stratio/Spark-MongoDB).

Let me know if you're open to deleting the old stuff and I can submit a PR, thanks!

**Comment by zero323** [comment]
Yeah, we should probably remove these and a few more. I am not sure what is the best approach here though.

When we discussed removals for the first time I imagined that we'll create another file (let's say REMOVED.md) and put things in there, with some standardized description of the removal reason. This would clearly indicate that the project was listed, but has been removed (and possibly indicate when it might be added back, i.e. if project is resurrected). 

I'd also prefer to remove things one at the time  ‒ it's easier to track things this way and keep any discussion focused.

**Comment by eliasah** [comment]
I agree with @zero323 . It would be better to keep track maybe in a separate file.

**Comment by zero323** [comment]
@eliasah Now we [have one](https://github.com/awesome-spark/awesome-spark/blob/main/DEPRECATED.md), it is just a matter of making a sweep, though there are some things that can go directly into trash bin I guess (like Berkley MOOCs, which are not even searchable on edx anymore).

**Comment by holdenk** [comment]
Apache Bahir is also probably good to be removed (it is in the Apache Attic)
